app/views/portals/thread-id.vue

```diff
@@ -1,181 +1,135 @@
-            <div v-if="forum && forum.isClosed"><em>{{ $t('forumIsClosedForComments') }}</em></div>  
-            <div v-else-if="thread.isClosed"><em>{{ $t('threadIsClosedForComments') }}</em></div>  
+            <div v-if="forum?.isClosed"><em>{{ t('forumIsClosedForComments') }}</em></div>
+            <div v-else-if="thread?.isClosed"><em>{{ t('threadIsClosedForComments') }}</em></div>
           </div>
-
           <div class="col-auto align-self-center">
-
             <button v-if="isOpen && subscription" :disabled="subscribing"
-              class="btn btn-sm" :class="{ 'btn-outline-dark':!subscription.watching, 'btn-dark': subscription.watching }" type="button"
-              @click="toggleSubscription()">
-              <span v-if="subscription.watching"><i class="fa fa-envelope-o"></i> {{ $t('buttonUnsubscribe') }} </span>
-              <span v-else><i class="fa fa-envelope-o"></i> {{ $t('buttonSubscribe') }} </span>
+              class="btn btn-sm" :class="{ 'btn-outline-dark': !subscription.watching, 'btn-dark': subscription.watching }" type="button"
+              @click="toggleSubscription">
+              <span v-if="subscription.watching"><i class="fa fa-envelope-o"></i> {{ t('buttonUnsubscribe') }} </span>
+              <span v-else><i class="fa fa-envelope-o"></i> {{ t('buttonSubscribe') }} </span>
               <i v-if="subscribing" class="fa fa-cog fa-spin"></i>
             </button>
-
           </div>
         </div>
       </div>
-
-      <h1>{{ lstring(thread.subject) }}</h1> 
-
-      <post :post="thread" class="mb-2" @refresh="refresh()">
+      <h1>{{ lstring(thread.subject) }}</h1>
+      <post :post="thread" class="mb-2" @refresh="load">
         <template v-slot:showReplies="{ replies }">
-            <em>
-              <a class="anchor-margin" name="replies"></a>
-              <span v-if="replies == 0"> {{ $t('linkNoReplies', { count: replies }) }}</span>
-              <span v-if="replies == 1"><i class="fa fa-comment"></i> {{ $t('linkOneReply', { count: replies }) }} </span>
-              <span v-if="replies  > 1"><i class="fa fa-comments"></i> {{ $t('linkXReplies', { count: replies }) }} </span>
-            </em>
+          <em>
+            <a class="anchor-margin" name="replies"></a>
+            <span v-if="replies == 0"> {{ t('linkNoReplies', { count: replies }) }}</span>
+            <span v-if="replies == 1"><i class="fa fa-comment"></i> {{ t('linkOneReply', { count: replies }) }} </span>
+            <span v-if="replies  > 1"><i class="fa fa-comments"></i> {{ t('linkXReplies', { count: replies }) }} </span>
+          </em>
         </template>
       </post>
-
-
-      <div v-if="posts" >
-        <post class="mb-2 border-top" v-for="reply in posts" :key="reply.postId" :post="reply" @refresh="refresh()" />
+      <div v-if="posts">
+        <post class="mb-2 border-top" v-for="reply in posts" :key="reply.postId" :post="reply" @refresh="load" />
```

Comment:
preferable to use event with `()`
```suggestion
        <post class="mb-2 border-top" v-for="reply in posts" :key="reply.postId" :post="reply" @refresh="load()" />
```